src/components/atoms/SelectButton.tsx

```diff
@@ -58,10 +60,10 @@ const SelectButton: FC<IProps> = ({ onChange, initialValue }) => {
             value={selectValue}
             onChange={handleChange}
           >
-            <MenuItem value={1}>שנה אחרונה</MenuItem>
-            <MenuItem value={3}>3 שנים אחרונות</MenuItem>
-            <MenuItem value={5}>5 שנים אחרונות</MenuItem>
-            <MenuItem value={8}>8 שנים אחרונות</MenuItem>
+            <MenuItem value={1}>{t('filterBar.yearChoices.0')}</MenuItem>
```

Comment:
I think using an object here will be better in the long run.
You can do it in a seperate PR if you like